cmd/server-test/error_analyzer_test.go

```diff
@@ -72,9 +50,13 @@ func (suite *ErrorAnalyzerIntegrationSuite) TearDownTest() {
 func (suite *ErrorAnalyzerIntegrationSuite) TestAnalyzerErr() {
 	suite.sendEvent(successJSON)
 
-	suite.GrepTrue(suite.r, `msg="analysis failed" analyzer=Dummy app=lookoutd error="rpc error: code = Unknown desc = review error"`)
+	suite.GrepTrue(suite.r, suite.errMessage)
 }
 
 func TestErrorAnalyzerIntegrationSuite(t *testing.T) {
-	suite.Run(t, new(ErrorAnalyzerIntegrationSuite))
+	suite.Run(t, &ErrorAnalyzerIntegrationSuite{
```

Comment:
Now that `ErrorAnalyzerIntegrationSuite` and `TimeoutErrorAnalyzerIntegrationSuite` are different types maybe we could revert the changes for the new fields (`configFile`, `analyzer`, `errMessage`), and call `suite.Run(t, new(ErrorAnalyzerIntegrationSuite))` here.